examples/example1.jl

```diff
@@ -16,12 +25,14 @@ m = Model()
 
 @consumer(m, RA, benchmark = 150.)
 
-@production(m, X, 0, 1, [Output(PX, 100)], [Input(PL, 50), Input(PK, 50)])
-@production(m, Y,0, 1, [Output(PY, 50)], [Input(PL, 20), Input(PK, 30)])
-@production(m, U, 0, 1, [Output(PU, 150)], [Input(PX, 100), Input(PY, 50)])
+@production(m, X, :($transf_elas_x*1.), :($sub_elas_x*1.), [Output(PX, 100)], [Input(PL, 50), Input(PK, 50)])
+@production(m, Y, :($transf_elas_y*1.), :($sub_elas_y*1.), [Output(PY, 50)], [Input(PL, 20), Input(PK, 30)])
+@production(m, U, 0, :($sub_elas_u*1.), [Output(PU, 150)], [Input(PX, 100), Input(PY, 50)])
 
 @demand(m, RA, 1., [Demand(PU, 150)], [Endowment(PL, :(70 * $endow)), Endowment(PK, 80.)])
 
+# θXiPL = add!(m, ShareParameter(Symbol("θXiPL"), value=.5))
```

Comment:
Should this be here?

Reply:
No. More scratch/experimenting. I'll delete it.